build/misc.tcl

```diff
@@ -1812,21 +1712,21 @@ respond "\n" "TIME MACHIN\r"
 expect ":KILL"
 
 # 11LOGO
-respond "*" ":cwd 11logo\r"
-respond "*" ":plx143 /H/M/CL BIN,N CREF_SYSTEM,TYI,READ,EVAL,TURTLE,ZEND\r"
-respond "ASSSW=" "0\r"
-expect ":KILL"
+cwd "11logo"
+palx "/H/M/CL BIN,N CREF" "SYSTEM,TYI,READ,EVAL,TURTLE,ZEND" {
```

Comment:
Are you sure it wasn’t clearer that there were two distinct versions of PALX (the one used by 11logo being the 143 version) when the one used by 11logo was called PLX143? Now they both are called PALX.